src/languages/javascript.coffee

```diff
@@ -15,14 +15,17 @@ module.exports = {
   Supported Grammars
   ###
   grammars: [
-    "JavaScript"
+    "JavaScript",
+    "JavaScript for Automation (JXA)"
   ]
 
   ###
   Supported extensions
   ###
   extensions: [
-    "js"
+    "js",
+    "scpt",
+    "applescript"
```

Comment:
I don't think this should work on _.applescript_ files.
[Here](https://github.com/danielbayley/atom-language-javascript-jxa/blob/master/grammars/jxa.cson#L3-L7) are the `fileTypes:` I have in my _language-javascript-jxa_ package… so this should probably match those.